lib/Doctrine/Common/Inflector/Inflector.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Not possible in stable branch.

Reply:
Why? :(

Reply:
Yeah, [we can't](https://secure.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration.strict).

Reverted, now, RTM :)